packages/carbon-web-components/package.json

```diff
@@ -70,8 +70,10 @@
     "@babel/runtime": "^7.16.3",
     "@carbon/styles": "1.47.0",
     "flatpickr": "4.6.1",
+    "gulp-minify-html-literals": "^1.1.22",
```

Comment:
This package doesn't seem to exist. Is [this](https://www.npmjs.com/package/minify-html-literals) the one you meant to include?